assets/package.json

```diff
@@ -9,7 +9,8 @@
   },
   "dependencies": {
     "phoenix": "file:../deps/phoenix",
-    "phoenix_html": "file:../deps/phoenix_html"
+    "phoenix_html": "file:../deps/phoenix_html",
+    "throttle-debounce": "^2.0.0"
```

Comment:
I'd really be more comfortable using Lodash for this. We're probably going to end up with it one way or another down the line anyway, and I'd prefer a more mature library wherever possible. Plus there is official tooling for Lodash + Babel + Webpack to automatically cherry-pick and/or tree shake so we only end up with the necessary functions in production anyway. The primary one is `babel-plugin-lodash`.

Reply:
I agree, but I'm super uncomfortable doing anything like this. Can I open an issue and let you tackle it later?

Reply:
Sure